### PR TITLE
Test for Routes.jsx

### PR DESCRIPTION
**Pull request by kebrahim**
Works on #155 

- [x] Tests pass
- [x] Appropriate changes to README are included in PR
- [x] Feature has been fleshed out in design document

### CODE REVIEW

File: callmeback/frontend/src/Routes.jsx [hunk 1]
```diff
@@ -9,19 +9,17 @@ import CancelConfirmation from './components/CancelConfirmation.jsx';
...
-        <Switch>
-          <Route path='/home' component={Home} />
-          <Route path='/thankyou' component={ThankYou} />
-          <Route path='/cancel' component={Cancel} />
-          <Route path='/cancelconfirmation' component={CancelConfirmation} />
-          <Route path='/reservations/:id' component={Reservation} />
-          <Route exact path='/' component={Home} />
-        </Switch>
-      </div>
-    </Router>
```
Comment (mscharfstein) [inline callmeback/frontend/src/Routes.jsx:24]: why did you move the Router to index.js?

File: callmeback/frontend/src/components/Cancel.jsx [hunk 2]
```diff
@@ -8,6 +8,17 @@ import axios from 'axios';
...
+
+  if (!location || !location.state) {
+    return (
+      <Container
+        text
+        className='paper'
+        data-testid='cancel-container'
+      ></Container>
+    );
+  }
```
Comment (mscharfstein) [inline callmeback/frontend/src/components/Cancel.jsx:21]: i think this should be merged into the other return. like inside the main return you can have {!location || !location.state && <Container
        text
        className='paper'
        data-testid='cancel-container'
      ></Container>}
{!!location.state &&
<Container text className='paper' data-testid='cancel-container'> ...} or something

Comment (kebrahim) [reply callmeback/frontend/src/components/Cancel.jsx]: done (as best i could)

File: callmeback/frontend/src/Routes.jsx (hunk 1 above)
Comment (kebrahim) [reply callmeback/frontend/src/Routes.jsx:24]: i wasn't able to easily unit test it (& pass in the appropriate routes in each test case) when the Router was already specified in Routes.jsx
